cmd/user/internal/domain/user/commands.go

```diff
@@ -63,7 +84,9 @@ func NewCommandFromPayload(contract string, payload []byte) (domain.Command, err
 
 // RequestAccessToken command
 type RequestAccessToken struct {
-	ID uuid.UUID `json:"id"`
+	ID       uuid.UUID `json:"id"`
+	Email    string    `json:"email"`
+	Password string    `json:"password"`
```

Comment:
we should have password and salt

Reply:
Implemented in: 45ebb3a68db284c09d4c262b707b408f2498fd59